src/main/java/com/alexlasota/medical_clinic_proxy/controller/ProxyVisitController.java

```diff
@@ -0,0 +1,39 @@
+package com.alexlasota.medical_clinic_proxy.controller;
+
+import com.alexlasota.medical_clinic_proxy.model.VisitDto;
+import com.alexlasota.medical_clinic_proxy.service.ProxyVisitService;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping("/proxy/visits")
+public class ProxyVisitController {
+
+    private final ProxyVisitService proxyVisitService;
+
+    public ProxyVisitController(ProxyVisitService proxyVisitService) {
+        this.proxyVisitService = proxyVisitService;
+    }
+
+    @GetMapping("/patient/{patientId}")
+    public List<VisitDto> getVisitsByPatientId(@PathVariable Long patientId) {
+        return proxyVisitService.getVisitsByPatientId(patientId);
+    }
+
+    @RequestMapping(value = "/visits/{visitId}/patients/{patientId}", method = RequestMethod.PATCH)
```

Comment:
@PatchMapping

Reply:
RequestMethod.PATCH
jest to patch mapping ale zawarty w @RequestMapping ze wzgledu na feigna